polling_stations/apps/data_collection/management/commands/import_newham.py

```diff
@@ -3,56 +3,54 @@
 
 class Command(BaseXpressDemocracyClubCsvImporter):
     council_id = "E09000025"
-    addresses_name = "europarl.2019-05-23/Version 1/Democracy_Club__23May2019Newham.tsv"
-    stations_name = "europarl.2019-05-23/Version 1/Democracy_Club__23May2019Newham.tsv"
-    elections = ["europarl.2019-05-23"]
+    addresses_name = (
+        "parl.2019-12-12/Version 1/LBNewhamDemocracy_Club__12December2019.TSV"
+    )
+    stations_name = (
+        "parl.2019-12-12/Version 1/LBNewhamDemocracy_Club__12December2019.TSV"
+    )
+    elections = ["parl.2019-12-12"]
     csv_delimiter = "\t"
-
-    def station_record_to_dict(self, record):
-        # Fix from: 890908:polling_stations/apps/data_collection/management/commands/misc_fixes.py:224
-        if record.polling_place_id == "4483":
-            record = record._replace(polling_place_easting="538526.11")
-            record = record._replace(polling_place_northing="184252.81")
```

Comment:
Carpenters and Docklands Centre is in use again this election (id 4823). Given we've got an exact point, we might as well use it again.

Reply:
I don't think my git skills were up to par when I first tried to dereference the comment. I'll add it back in :-)